### DISCUSSION TITLE
How can get web cookie after october, 2023

### DISCUSSION THREAD

**Discussion by belikr** [SteamRE/SteamKit#1275] [General]
steam web api is not working after october, 2023

then 

how can i get cookie for access steam web?

**Comment by salat-23** [comment]
> steam web api is not working after october, 2023
> 
> then
> 
> how can i get cookie for access steam web?

bump

**Comment by SeRi0uS007** [comment]
https://github.com/SteamRE/SteamKit/issues/1125
https://github.com/fcastrocs/steam-web

You can explore how it works now.

**Comment by Christian-Stieber** [comment]
This is the general idea: https://github.com/SteamRE/SteamKit/commit/c3f37c08024bf8a11314f3a7e3998bef3b36ad31#commitcomment-131030406

**Reply by w4559wq** [reply]
But this solution cannot be applied to sentry file

**Reply by Christian-Stieber** [reply]
Sentry files have been used for SteamGuard, so it's an entirely different thing.

Also, sentry files are no longer used by Steam. You can still login (probably -- I haven't tried it ever since I noticed that they stopped working and updated my code to the new login system), but you'll have to do the SteamGuard code every time since it will no longer send you sentry data.

**Reply by w4559wq** [reply]
thanks

**Reply by w4559wq** [reply]
I just noticed that there is no way to obtain the refresh token for the account logged in through the sentinel file.

**Comment by xPaw** [comment]
There's a SteamKit sample here: https://github.com/SteamRE/SteamKit/blob/master/Samples/1c.WebCookie/Program.cs

TL;DR: Use new auth system, access token can be used as the web cookie.
